tests/bin/sgx_get_att_quote.c

```diff
@@ -5,14 +5,11 @@
  * Quote from get_attestation() and check that the first bytes of
  * the returned Quote in buf match expected values. */
 
-// TODO: Update these to match real Quote values.
-// See https://github.com/enarx/enarx-keepldr/issues/92.
-const unsigned char expected[512] = { 44, 44, 44 };
+// TODO: Determine expected Quote value
```

Comment:
I guess this value would change every time we change the shim-sgx, right?

Reply:
Yes, it would. So I don't think it would make sense to check it in its entirety, at least until the shim is very stable. However, there may be some values that are always the same (I think some QE data may be embedded in each Quote). I plan to review that and see if maybe we can just check those values, if they exist.